streams/sv.m3u

```diff
@@ -23,6 +23,8 @@ https://stream.oursnetworktv.com/latin/encoder44/playlist.m3u8
 https://5d32e2b9b7eed.streamlock.net:4443/ectv/ectv/playlist.m3u8
 #EXTINF:-1 tvg-id="ElimTV.sv",Elim TV (480p)
 https://dacastmmd.mmdlive.lldns.net/dacastmmd/7e84ed301b194b7b81e86f85bb2c3a33/playlist.m3u8
+#EXTINF:-1 tvg-id="FTVTelevisionFamiliar.sv",FTV Televisión Familiar (480p) [Not 24/7]
+https://www.yupi.live:1936/tvftv/tvftv/playlist.m3u8
```

Comment:
Seems to be broadcasting TVX.sv
![image](https://github.com/iptv-org/iptv/assets/5430116/c5a40f9d-8c61-49b5-9462-0652d1e004bb)

Reply:
Some channels do this, but FTV is a separate channel from TVX.